### PR TITLE
Set up CI to run e2e tests

### PR DESCRIPTION
**Pull request by SagarGi**
### Description
This PR set up the CI to run the web UI tests.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+      - name: set up go
+        uses: actions/setup-go@v4
+        with:
+          go-version: "1.20"
+
+      - name: check  node and go
+        run: |
+          node --version
+          go version
```
Comment (nabim777) [inline .github/workflows/ci.yml]: do we need this job also?

Comment (nabim777) [inline .github/workflows/ci.yml]: do we need this job also?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+      - name: set up the file application
+        run: |
+          git clone https://github.com/filebrowser/filebrowser
+          ls
+          cd filebrowser
+          cd frontend
```
Comment (nabim777) [inline .github/workflows/ci.yml]:
```suggestion
          cd filebrowser/frontend
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+          go-version: "1.20"
+
+      - name: check  node and go
+        run: |
+          node --version
+          go version
+
+      - name: set up the file application
+        run: |
+          git clone https://github.com/filebrowser/filebrowser
```
Comment (nabim777) [inline .github/workflows/ci.yml:30]: Is this needed &&
```suggestion
          git clone https://github.com/filebrowser/filebrowser &&
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+          ls
+          cd filebrowser
+          cd frontend
+          npm install
+          npm run build
+          cd ..
+          go mod download
```
Comment (nabim777) [inline .github/workflows/ci.yml:35]:
```suggestion
          npm install &&
          npm run build &&
          cd .. &&
          go mod download &&
```

**Review by SoniGautam — APPROVED** [review]

**Review by S-Panta — APPROVED** [review]

**Pull request merged**